config.json

```diff
@@ -395,7 +395,13 @@
       "core": false,
       "difficulty": 5,
       "slug": "pascals-triangle",
-      "topics": null,
+      "topics": [
+	 "recursion",
+	 "arrays",
+	 "matrices",
+	 "algorithms",
+	 "mathematics"
```

Comment:
Great suggestions! :) The only one I'm not sure about is `recursion`. You could quite easily solve this exercise without recursion I think, and I would hesitate to encourage people to use recusion in Java when they don't need to since using iteration is more suited to the language. What do you think? :)

I would also add `exception_handling` and `integers` as topics since the tests expect an exception to be thrown and it uses integers. Do you agree? :)

Also, all of these should be indented two spaces in from `topics`, like the entries for any other exercise with topics :)

Reply:
I am fine with your suggestions. I will add `exception_handling` and `integers` as well. I guess it depends on whether you want the topic list to be more on the exhaustive side or not. I was even considering adding `loops` :)